data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+			`	You follow the prefect into the hall, and discover that it is one of the many cafeterias that seem to be common in Remnant starports. He picks up two trays from the window and leads you to an empty table.`
+			choice
+				`	"What secret do you want me to keep?`
+					goto secret
+				`	"What constitutes 'things that influence humanity's survival?"`
+			`	He takes a bite before responding. "Basically, anything that might either pose an existential threat to humanity; or anything that we might learn from and put to use to ensure humanity's survival. It might be an invading fleet of automated machines, or a spaceborn plague. But it is just as possible you might find beings that we can work alongside to form strong partnerships, or ancient ruins with old technology that points us towards new discoveries. The possibilities are indeed infinite.`
+				goto feast
+			label secret
+			`	He sips on thick drink before responding. "Quite simply, the fact that we are here. To the best of our knowledge, only the Drak and the Korath know that we exist, and of those only the Korath have raided our worlds. Our intention is to the last bastion of humanity, and we rely on secrecy to achieve that goal." He pauses for a moment to eat, then continues. "The Alphas are a fearsome threat, but they may be the least of humanity's worries in the grand scheme of things. Thus we strive to maintain whatever advantages we can, and one of those is surprise. If no-one knows we are here, we have the advantage over any that might seek to conquer or destroy us."`
+			`	"Of course, maintaining the secrecy of our capabilities is also very important - and the reason why you had to prove yourself before we let you purchase our ships and outfits - but it is secondary to the fundemental secrecy of our existence. Ideally, no-one should know we exist. Those who do know, should ideally think of us as little more than refugees struggling to survive in a hostile environment`
+			label feast
+			`	He waves dismisively. "I am sure you have more questions, but let us leave it there for now. Our meals await, and, well, I need to show you some stuff in regards to our security procedures. So finish your meal, and let's head over to <destination>."`
+				accept
+	on enter
+		conversation
+			`Once you reach space, the prefect interrupts your normal routine. "I'm sure you noticed that your 'license' isn't exactly a normal ID card. There's a reason for that, namely that it has a whole bunch of extra circuitry in it, and includes a couple sensors on the surface. I don't understand everything in it, but the security specialists assure me that it is incredibly resistance to being hacked or compromised."`
```

Comment:
When did the player physically get the license?

Reply:
There is now a section near the start where the fellow actually gives you the license. (and the player is shown the license)